chrome/sunset.js

```diff
@@ -158,6 +157,32 @@ Sunset.showNotification_ = function(index) {
   chrome.storage.sync.set({
     "index": index + 1
   });
+
+  // Uninstall the extension a minute after the last notification.
+  if (index >= Sunset.timeline_offsets_.length - 1) {
+    chrome.alarms.create("uninstall", {
+        "delayInMinutes": 1
+    });
+  }
+}
+
+
+/**
+ * Handles alarms.
+ *
+ * @param {Alarm} alarm The alarm that fired.
+ * @private
+ */
+Sunset.onAlarm_ = function(alarm) {
+  // Consider showing a notification on the regular clock alarm.
+  if (alarm.name == "clock")
+    Sunset.maybeShowNotification_();
+
+  // Uninstall the extension and show an article on the uninstall delay alarm.
+  if (alarm.name == "uninstall") {
+    Sunset.showArticle_();
```

Comment:
Leave this out for the moment, as I think we haven't reached a decision about whether or not it's the right thing to do.